core/src/Revolution/modLexicon.php

```diff
@@ -186,6 +186,7 @@ public function load()
         $topics = func_get_args(); /* allow for dynamic number of lexicons to load */
 
         if ($this->modx->context && $this->modx->context->get('key') == 'mgr') {
+            $_SESSION = isset($_SESSION) ? $_SESSION : [];
             $defaultLanguage = $this->modx->getOption('manager_language', $_SESSION,
```

Comment:
```suggestion
            $defaultLanguage = $this->modx->getOption('manager_language', $_SESSION ?? [],
```

And check above is not needed.